src/directory/person.css

```diff
@@ -0,0 +1,49 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
```

Comment:
@Alihaider168 we dont need to apply css on body from within a non-layout container.